aws/resource_aws_transfer_user_test.go

```diff
@@ -43,6 +43,38 @@ func TestAccAWSTransferUser_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSTransferUser_restricted(t *testing.T) {
+	var conf transfer.DescribedUser
+	rName := acctest.RandString(10)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheck(t); testAccPreCheckAWSTransfer(t) },
+		IDRefreshName: "aws_transfer_user.foo",
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckAWSTransferUserDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSTransferUserConfig_restricted(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSTransferUserExists("aws_transfer_user.foo", &conf),
```

Comment:
add resourceName := "aws_transfer_user.foo" and replace all "aws_transfer_user.foo" occurrences